daemon/debugtrap_windows.go

```diff
@@ -0,0 +1,30 @@
+package daemon
```

Comment:
It's not required as _windows is sufficient. (Although !windows is required in the case of _unix files as _unix isn't a golang build directive).